tasks/Sequential dyadic die-rolling.md

```diff
@@ -0,0 +1,112 @@
+# Summary
+> This study examines the the role of collaboration in shaping corruption through a sequential dyadic dice-rolling game. In a incentive structure based on rolling doubles, players in the collaborative enviornment reported signficantly more doubles than when players played alone and when lying only benefited one party. 
+
+
+# References
+> Appendix File includes instructions and computer screen reference images: https://www.pnas.org/content/pnas/suppl/2015/08/05/1423035112.DCSupplemental/pnas.1423035112.sapp.pdf
+
+# Stimuli
+## The visual components
+> N/A
+
+## Materials for alternative versions of the experiment 
+> N/A
+
+# Procedure
+## Steps
+> Step One: The general instructions, which were read out loud by an experimenter, informed participants that they will engage in a study composed of three stages. Each stage was explained separately once the previous stage was completed.
+
+> Step Two: Participants learned that they will earn money based on their performance in each of the three stages. 
+
+> Step Three: In the first stage, participants learned that they will engage in a die-rolling task in which the amount of money they will earn depends on the results of two die rolls. Following the general instructions, participants received on-screen treatment-specific information about the rules determining their payoff and had a few practice rounds to make sure they understood the payoffs associated with each roll. 
+
+> Step Four: After indicating they understood rules, participants were paired up and played one period of die rolling task for payoff 
+
+> Step Five: Participants were told that in the second stage they will engage in up to 30 additional periods with a different partner, and that they will be paid according to the results of one of these periods, which will be randomly selected. In reality there were always 20 additional periods and the participants payoff was varried based on experimental treatments (two person treatments vs. one person treatment) The payment incentives varied between treatments, with different inventives for A and B in a series of treatments.
+
+> Step Six: Phase three focused o social value orientation through use of SVO slider. The measure assesses magnitude of people's concern to others' outcomes. Participants made a series of 15 choices between 9 resource allocation opporunities between self and another participant 
+
```

Comment:
It might be worth mentioning that they have the choice of telling the truth or lying.

Also I usually write my procedure as what the person actively does, so for this it would be:

1. receives instructions regarding how to play and earn payoff
2. engages in practices rounds
3. gets paired with partner
4. Player A rolls and reports result to player B
5. Player B rolls and reports result to player A
6. If the answers match, both players receive payoff
7. Repeat steps 4-6 up to 30 times

I'm not sure if which way of doing it is correct and might be worth clarifying